src/contexts/OracleDataContext.tsx

```diff
@@ -195,6 +196,10 @@ export function OracleDataProvider({ children }: { children: ReactNode }) {
         dispatch({ type: "assertions", data: assertions }),
       errors: setErrors,
     });
+    oracleEthersApiList.map(
```

Comment:
this is a test. super important we never do this unless the graph call is missing for this chain/oracle.  that is not the easiest thing to figure out cuz the configs are so messy